data/sql_updates/create_totals_presidential_view.sql

```diff
@@ -42,7 +42,8 @@ from
     left join dimdates start_date on cvg_start_dt_sk = start_date.date_sk and cvg_start_dt_sk != 1
     left join dimdates end_date on cvg_end_dt_sk = end_date.date_sk and cvg_end_dt_sk != 1
 where
-    p.expire_date is null or p.expire_date > date_trunc('day', now())
+    (p.expire_date is null or p.expire_date > date_trunc('day', now()))
```

Comment:
For the fact tables, If something has an expire date at all, that means that it has been amended. Since we are looking for totals, we don't want to add up the expired records. 

I don't think this is wrong, but you could just check if the expire_date is null, if you wanted to. This would apply to all the totals tables.